src/patternfly/components/Form/form.scss

```diff
@@ -106,6 +110,16 @@
   }
 }
 
+.pf-c-form__section {
+  display: grid;
+  gap: var(--pf-c-form__section--Gap);
+
+  & + .pf-c-form__group:not(.pf-m-action),
+  & ~ & {
```

Comment:
is the expectation that if you have a regular form group followed by a section, that you would get the larger section spacing? if so, that isn't working currently. I think you would replace `& ~ &` with `&:not(:first-child)`?